4.x: issues with legacy maven archetypes

## Environment Details
* Helidon Version: 4.0.0-M2

----------

## Problem Description

There appear to be some problems with the legacy maven archetypes in 4.x.

1.  The SE archetypes appear to be missing
2. When executing one of the MP archetypes I get an error

```
mvn -U archetype:generate -DinteractiveMode=false -DarchetypeGroupId=io.helidon.archetypes  \
-DarchetypeArtifactId=helidon-quickstart-mp -DarchetypeVersion=4.0.0-M2 -DgroupId=io.helidon.examples  \
-DartifactId=helidon-quickstart-mp -Dpackage=io.helidon.examples.quickstart.mp -Possrh-staging
. . .

﻿[ERROR] Failed to execute goal org.apache.maven.plugins:maven-archetype-plugin:3.2.1:generate (default-cli) 
on project standalone-pom: Execution default-cli of goal org.apache.maven.plugins:maven-archetype-plugin:3.2.1:generate failed: 
A required class was missing while executing org.apache.maven.plugins:maven-archetype-plugin:3.2.1:generate: 
io/helidon/build/archetype/engine/v2/ArchetypeEngineV2$Builder
```

--- a/applications/parent/pom.xml
+++ b/applications/parent/pom.xml
@@ -42,8 +42,8 @@
         <version.plugin.dependency>3.6.0</version.plugin.dependency>
         <version.plugin.exec>3.1.0</version.plugin.exec>
         <version.plugin.failsafe>3.1.2</version.plugin.failsafe>
-        <version.plugin.helidon>4.0.0-M1</version.plugin.helidon>
-        <version.plugin.helidon-cli>4.0.0-M1</version.plugin.helidon-cli>
+        <version.plugin.helidon>4.0.0-M2</version.plugin.helidon>
+        <version.plugin.helidon-cli>4.0.0-M2</version.plugin.helidon-cli>
         <version.plugin.jar>3.0.2</version.plugin.jar>
         <version.plugin.nativeimage>0.9.16</version.plugin.nativeimage>
         <version.plugin.os>1.5.0.Final</version.plugin.os>

--- a/examples/quickstarts/helidon-standalone-quickstart-mp/pom.xml
+++ b/examples/quickstarts/helidon-standalone-quickstart-mp/pom.xml
@@ -41,8 +41,8 @@
         <version.plugin.eclipselink>2.7.5.1</version.plugin.eclipselink>
         <version.plugin.exec>1.6.0</version.plugin.exec>
         <version.plugin.failsafe>3.0.0-M5</version.plugin.failsafe>
-        <version.plugin.helidon>4.0.0-M1</version.plugin.helidon>
-        <version.plugin.helidon-cli>4.0.0-M1</version.plugin.helidon-cli>
+        <version.plugin.helidon>4.0.0-M2</version.plugin.helidon>
+        <version.plugin.helidon-cli>4.0.0-M2</version.plugin.helidon-cli>
         <version.plugin.jandex>3.1.2</version.plugin.jandex>
         <version.plugin.jar>3.0.2</version.plugin.jar>
         <version.plugin.os>1.5.0.Final</version.plugin.os>

--- a/examples/quickstarts/helidon-standalone-quickstart-se/pom.xml
+++ b/examples/quickstarts/helidon-standalone-quickstart-se/pom.xml
@@ -41,8 +41,8 @@
         <version.plugin.dependency>3.6.0</version.plugin.dependency>
         <version.plugin.exec>1.6.0</version.plugin.exec>
         <version.plugin.failsafe>3.0.0-M5</version.plugin.failsafe>
-        <version.plugin.helidon>4.0.0-M1</version.plugin.helidon>
-        <version.plugin.helidon-cli>4.0.0-M1</version.plugin.helidon-cli>
+        <version.plugin.helidon>4.0.0-M2</version.plugin.helidon>
+        <version.plugin.helidon-cli>4.0.0-M2</version.plugin.helidon-cli>
         <version.plugin.jar>3.0.2</version.plugin.jar>
         <version.plugin.os>1.5.0.Final</version.plugin.os>
         <version.plugin.protobuf>0.5.1</version.plugin.protobuf>

--- a/pom.xml
+++ b/pom.xml
@@ -108,7 +108,7 @@
         <version.plugin.exec>3.1.0</version.plugin.exec>
         <version.plugin.failsafe>3.1.2</version.plugin.failsafe>
         <version.plugin.glassfish-copyright>2.3</version.plugin.glassfish-copyright>
-        <version.plugin.helidon-build-tools>4.0.0-M1</version.plugin.helidon-build-tools>
+        <version.plugin.helidon-build-tools>4.0.0-M2</version.plugin.helidon-build-tools>
         <version.plugin.hibernate-enhance>${version.lib.hibernate}</version.plugin.hibernate-enhance>
         <version.plugin.jacoco>0.8.5</version.plugin.jacoco>
         <version.plugin.jandex>3.1.2</version.plugin.jandex>

